plugins/index.js

```diff
@@ -336,5 +504,9 @@ module.exports = (on, config) => {
     config.env.SKIP_METAMASK_SETUP = true;
   }
 
+  if (process.env.PROVIDERS) {
+    config.env.PROVIDERS = process.env.PROVIDERS;
+  }
+
```

Comment:
Same question. Why do we set it up here?

Reply:
will remove